_sources/WorldFacts/cs2_graphing_business_data.rst

```diff
@@ -450,12 +450,14 @@ above and apply that to the world below.
 
 
 .. image:: Figures/WorldFactbook_74_0.png
+:alt: This is the world map without color. 
```

Comment:
Indentation still needs fixing

Reply:
BTW, building the book is a good way to catch all of these, as the bad indentations will cause errors.  Thanks.

Reply:
We are working on building the book with the fixed indentation before we send in a new pull request. We are sorry that this is taking long.